aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/UpdateHandler.java

```diff
@@ -27,11 +33,12 @@
         final CallbackContext callbackContext,
         final Logger logger) {
 
-        // Everything except RetentionPolicyInDays and KmsKeyId is createOnly
+        // Everything except RetentionPolicyInDays, KmsKeyId and Tags is createOnly
```

Comment:
we probably shouldn't maintain this list in multiple places
```suggestion
```
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/0406d2bd4e31329bfb2b8a10b97a21cebeaf4d79/aws-logs-loggroup/aws-logs-loggroup.json#L121-L123